jcommune-service/src/main/java/org/jtalks/jcommune/service/GroupService.java

```diff
@@ -53,7 +54,7 @@
      */
     List<Group> getByName(String name);
 
-    List<UserDto> getGroupUsers(long id, int count);
+    Page<UserDto> getPagedGroupUsers(long id, String page, int pageSize);
```

Comment:
Why is page a string? I'd assume that controllers convert it to integer and pass that integer to the service..